### ISSUE TITLE
Setup in readme fails with undefined config property `imgurClientID`

### ISSUE THREAD

**Issue by AndrewPardoe** [KenEucker/biketag-website#124]
I did a naïve install/setup based on the readme and ran into this issue on both Ubuntu/WSL2 npm version 6.14.4 and MacOSX. I don't expect much action for a config error on my part, but it's good to have the issue recorded in case folks are running into this.

I'm not much of a node/js dev, and I really didn't try to debug this issue. Consider this as FYI and close if you want.

```text
ap@Studio:~/biketag-website$ npm install
ap@Studio:~/biketag-website$ cp config.sample.json config.json
ap@Studio:~/biketag-website$ npm run run

> biketag-website@0.3.0 run /home/ap/biketag-website
> node app.js

BikeTag MultiTenant WebApp InitSeq
/home/ap/biketag-website/app.js:59
                return !!tokens[name] ? tokens[name] : (!!config[name] ? config[name] : config.defaults[name])
                                                                                                       ^

TypeError: Cannot read property 'imgurClientID' of undefined
    at getValueFromConfig (/home/ap/biketag-website/app.js:59:90)
    at setVars (/home/ap/biketag-website/app.js:68:32)
    at Object.<anonymous> (/home/ap/biketag-website/app.js:985:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! biketag-website@0.3.0 run: `node app.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the biketag-website@0.3.0 run script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ap/.npm/_logs/2020-06-30T00_43_31_529Z-debug.log
```

**Comment by KenEucker** [comment]
@AndrewPardoe I've made some updates that should get you up and running again. Note the changes in the readme that take the line you used, `cp config.sample.json config.json`, and set it to the correct path of `cp config.sample.json config/config.json`. 

That should be about it to get you running now. You may also need to clear out node_modules and reinstall, using the command `rm -rf node_modules && npm install` once you have pulled in the most recent changes.

Let me know if that works for you?

**Comment by KenEucker** [comment]
@AndrewPardoe did you happen to see my comment above?

**Comment by AndrewPardoe** [comment]
Thanks, just did. I have some time this week. 

I'm now seeing 

```
ap@Studio:~/biketag-website$ npm run run
npm ERR! missing script: run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ap/.npm/_logs/2020-09-07T03_15_32_887Z-debug.log
ap@Studio:~/biketag-website$ npm run dev

> biketag-website@0.4.0 dev /home/ap/biketag-website
> node app.js -- --debug=true

internal/modules/cjs/loader.js:800
    throw err;
    ^

SyntaxError: /home/ap/biketag-website/config/config.json: Unexpected token } in JSON at position 1125
    at JSON.parse (<anonymous>)
    at Object.Module._extensions..json (internal/modules/cjs/loader.js:797:27)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at clobber (/home/ap/biketag-website/node_modules/clobfig/index.js:66:55)
    at Array.reduce (<anonymous>)
    at Clobfig.getConfigurationFilesData (/home/ap/biketag-website/node_modules/clobfig/index.js:80:63)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! biketag-website@0.4.0 dev: `node app.js -- --debug=true`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the biketag-website@0.4.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ap/.npm/_logs/2020-09-07T03_15_41_274Z-debug.log
```

I'll dig around a bit. 

Note that `npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):` fails to install, but Google implies that it's a Darwin-only package?

**Comment by AndrewPardoe** [comment]
This is odd

> /home/ap/biketag-website/config/config.json: Unexpected token } in JSON at position 1125

The JSON (which was just a cp from the sample JSON) appears to be well-formed. Maybe I should try this on my Mac again.

**Comment by AndrewPardoe** [comment]
Resolved by #137
